src/Avatar.jsx

```diff
@@ -0,0 +1,25 @@
+import './Avatar.css';
+
+const Avatar = ({sharedFolder}) => {
+  const isShowFolderInfos = sharedFolder && sharedFolder.owner && sharedFolder.name;
+
+  return (
+    <div className='container-header'>
+      <div className="header-frame_ly">
+        <div className='user_ly'>
+          {isShowFolderInfos ? (
+            <div className='avatar_ly'>
+              <img className="avatar-img" src={sharedFolder.owner.profileImageSource} alt={`${sharedFolder.owner.name} 아바타`}/>
+              <span className='avatar-name'>{sharedFolder.owner.name}</span>
+            </div>
+          ) : (
+            <button className='login-button'>회원가입</button>
```

Comment:
form을 submit하는 용도의 버튼이 아니라면 요렇게 명시적으로 표현해주는게 좋은 것 같아요~
```suggestion
            <button className='login-button' type='button'>회원가입</button>
```

Reply:
안녕하세요